src/core.mk

```diff
@@ -17,6 +17,8 @@
 # General source files
 GEN_CPU_SRC	= core/legate_c.cc                 \
 							core/comm/comm.cc                \
+							core/comm/comm_cpu.cc            \
+							core/comm/coll.cc				 \
```

Comment:
Please match the formatting

Reply:
Fixed. Somehow the format is OK in the VS code, and I can only see the wrong format in vim.